docs/send-data/kubernetes/collecting-application-metrics.md

```diff
@@ -0,0 +1,437 @@
+:::
+
+[sumologicschema]:
+  https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/processor/sumologicschemaprocessor#attribute-translation
+
+### Renaming metric
+
+In order to rename metrics, the [transformprocessor] can be use. Please look at the following snippet:
+
+```yaml
+sumologic:
+  metrics:
+    otelcol:
+      extraProcessors:
+        - transform/1:
+            metric_statements:
+              - context: metric
+                statements:
+                  ## Renames <old_name> to <new_name>
+                  - set(name, "<new_name>") where name == "<old_name>"
+```
+
+### Adding or renaming metadata
+
+If you want to add or rename metadata, the [transformprocessor] can be use. Please look at the following snippet:
+
+```yaml
+sumologic:
+  metrics:
+    otelcol:
+      extraProcessors:
+        - transform/1:
+            metric_statements:
+              - context: resource
+                statements:
+                  ## adds <new_name> metadata
+                  - set(attributes["<new_name>"], attributes["<old_name>"])
+                  ## adds <new_static_name> metadata
+                  - set(attributes["<new_static_name>"], "<static_value>")
+                  ## removes <old_name> metadata
+                  - delete_key(attributes, "<old_name>")
+```
+
+**Note:** See [Default attributes](#default-attributes) for more information about attributes.
+
+[transformprocessor]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor
+
+## Investigation
+
+If you do not see your metrics in Sumo Logic, please check the following stages:
+
+- [Collecting Application Metrics](collecting-application-metrics.md)
+  - [Scraping metrics](#scraping-metrics)
+    - [Application metrics are exposed (one endpoint scenario)](#application-metrics-are-exposed-one-endpoint-scenario)
+    - [Application metrics are exposed (multiple endpoints scenario)](#application-metrics-are-exposed-multiple-enpoints-scenario)
+      - [Example](#example)
+    - [Application metrics are not exposed](#application-metrics-are-not-exposed)
+  - [Metrics modifications](#metrics-modifications)
+    - [Filtering metrics](#filtering-metrics)
+      - [Default attributes](#default-attributes)
+    - [Renaming metric](#renaming-metric)
+    - [Adding or renaming metadata](#adding-or-renaming-metadata)
+  - [Investigation](#investigation)
+    - [Check if metrics are in Prometheus](#check-if-metrics-are-in-prometheus)
+      - [Investigate Prometheus scrape configuration](#investigate-prometheus-scrape-configuration)
+        - [Pod is visible in Prometheus targets](#pod-is-visible-in-prometheus-targets)
+        - [There is no target for serviceMonitor](#there-is-no-target-for-servicemonitor)
+        - [Pod is not visible in target for custom serviceMonitor](#pod-is-not-visible-in-target-for-custom-servicemonitor)
```

Comment:
I don't think we need this whole TOC here. Telling the user to make sure they followed the steps from the start of this document should be enough.

Reply:
Agreed.